Fix issue with double-well potentials

RFMalenda opened the pull request:
Fix issue with double-well potentials

RFMalenda committed:
Add files via upload

ndattani commented:
Thanks for the contribution @RFMalenda! 

Since you've added a new file rather than changing an existing file, I'm not able to easily see the changes [here](https://github.com/HPQC-LABS/LEVEL/pull/4/files).

I believe the  following sequence of commands will fix this if they are run from the LEVEL/LEVEL_2022 folder in your fork:
```
git pull
git mv levelAS_RMCF_DoubleFix.f levelAS.f
git commit -am "rename levelAS_RMCF_DoubleFix.f to levelAS.f"
git push
```

Let me know if it works!

RFMalenda committed:
rename levelAS_RMCF_DoubleFix.f to levelAS.f

rename levelAS_RMCF_DoubleFix.f to levelAS.f

ndattani commented:
Thanks @RFMalenda! Now we can easily see the differences between your version and the previous version, here: https://github.com/HPQC-LABS/LEVEL/pull/4/files

RFMalenda committed:
Print and read wave function changes

Fix the check on the correction terms for double wells
Writing wave functions as a function of RVB to either the .10 file if LPRW<0 and to individual .fin files if LPRW>0
Reading in wave functions from .fin files
Adjusted formatting for the .7 output file

ndattani commented:
@RFMalenda Thanks for adding the support for .fin files today! I wonder if you could add to the LEVEL_2022/test_suite folder, an example of how it works? When significant changes are made to the code, we want to make sure that the new version of the code passes all tests in the test suite.

RFMalenda committed:
Create README

Explanation of how to use write and read function to generate .fun files including wave function information and read from .fin files to get wave function information

RFMalenda committed:
Add files via upload

RFMalenda commented:
@ndattani We have added some test files including inputs and outputs, along with a README explaining the changes and files.  Please let me know if there is anything else we need to do or any questions we can answer.

RFMalenda committed:
Added YVB to write files for wave functions
